src/kit/RadioGroup.module.scss

```diff
@@ -29,15 +29,47 @@
     align-items: center;
     display: flex;
   }
-  .option > span > *:nth-child(2) {
-    margin-left: var(--spacing-md);
-  }
   .label {
     font-size: 14px;
   }
-}
-.base.iconOnly {
-  .option {
-    padding: 0 var(--spacing-sm);
+  &.iconOnly {
+    .option {
+      padding: 0 var(--spacing-sm);
+    }
+  }
+  &.row {
+    display: flex;
+    gap: 8px;
+    height: 80px;
+
+    > * {
+      flex-grow: 1;
+    }
+    .option {
+      border-color: var(--theme-surface-border);
+      border-inline-start: solid 1px var(--theme-surface-border);
+      border-radius: var(--theme-border-radius);
+
+      &:hover {
+        border-color: var(--theme-status-active-weak);
+      }
+      &:not(:first-child)::before {
```

Comment:
q: what is this selecting for?

Reply:
replacing `Radio.Button` with `Button` for "Row" variation and removing all of this CSS.